app/routes/meetings/components/MeetingDetail.tsx

```diff
@@ -190,6 +191,15 @@ const MeetingDetails = () => {
           <ul>
             {attendanceButtons(statusMe, meeting.startTime)}
             <InfoList items={infoItems} />
+            <li>
+              <AddToCalender
+                title={meeting.title}
+                startTime={meeting.startTime}
+                endTime={meeting.endTime}
+                description={meeting.description}
+                location={meeting.location}
```

Comment:
I think it's easier to only pass the `meeting` object as the prop and then destruct it in the `AddToCalender` component
```suggestion
                meeting={meeting}
```